src/Domain/LeanCode.CQRS.RemoteHttp.Server/BaseRemoteObjectHandler.cs

```diff
@@ -84,6 +84,11 @@ public async Task<ExecutionResult> ExecuteAsync(HttpContext context)
                     obj,
                     type);
             }
+            catch (TaskCanceledException ex)
```

Comment:
`OperationCanceledException`, `TaskCanceledException` is when the task _scheduling_ is canceled (that won't occur here).